Cargo.toml

```diff
@@ -65,7 +65,7 @@ serde_cbor = "0.11.2"
 serde_json = "1.0.108"
 serde_pack = { version = "1.1.2", package = "rmp-serde" }
 surrealdb = { path = "lib", features = ["protocol-http", "protocol-ws", "rustls"] }
-surrealml-core = { version = "0.0.3", optional = true}
+surrealml-core = { version = "0.0.6", optional = true}
```

Comment:
Why the bump here? The CI is currently pinned to Rust v1.74.1 so I'm not sure if this will work but I know for a fact that this doesn't work on Rust v1.75 with our current build pipeline.

Reply:
Reverting, but that was required to fix my build